app/assets/stylesheets/custom.css.scss

```diff
@@ -22,11 +22,16 @@ body {
   background-color:rgba(255,255,255,0.5);
 }
 
+li {
+  padding-top: 3px;
```

Comment:
Properties should be ordered list-style-type, padding-bottom, padding-right, padding-top